src/settings.rs

```diff
@@ -2066,4 +2314,72 @@ mod tests {
 
         assert_eq!(expected, default);
     }
+
+    #[meilisearch_test]
+    async fn test_get_separator_tokens(index: Index) {
+        let separator: Vec<&str> = vec![];
+        let res = index.get_separator_tokens().await.unwrap();
+
+        assert_eq!(separator, res);
+    }
+
+    #[meilisearch_test]
+    async fn test_set_separator_tokens(client: Client, index: Index) {
+        let expected: Vec<String> = vec!["#".to_string(), "@".to_string()];
+
+        let task_info = index
+            .set_separator_tokens(&vec!["@".to_string(), "#".to_string()])
+            .await
+            .unwrap();
+        client.wait_for_task(task_info, None, None).await.unwrap();
+
+        let mut res = index.get_separator_tokens().await.unwrap();
+        res.reverse();
+
+        assert_eq!(expected, res);
+    }
```

Comment:
```suggestion
    #[meilisearch_test]
    async fn test_set_separator_tokens(client: Client, index: Index) {
        let expected: Vec<String> = vec!["@".to_string(), "#".to_string()];

        let task_info = index
            .set_separator_tokens(&expected)
            .await
            .unwrap();
        client.wait_for_task(task_info, None, None).await.unwrap();

        let mut res = index.get_separator_tokens().await.unwrap();

        assert_eq!(expected, res);
    }
```

I believe we should be able to do that